test/BlazorSignalR.Test.Client/BlazorSignalR.Test.Client.csproj

```diff
@@ -1,14 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
+    <OutputType>Exe</OutputType>
     <RootNamespace>BlazorSignalR.Test.Client</RootNamespace>
-    <LangVersion>8.0</LangVersion> <!-- We need this, as the razor engine will use C# 8 features.-->
+    <LangVersion>8.0</LangVersion>
+    <!-- We need this, as the razor engine will use C# 8 features.-->
     <RazorLangVersion>3.0</RazorLangVersion>
+    <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.1.0-preview3.19555.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.1.0-preview3.19555.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.1.0-preview4.19579.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.1.0-preview4.19579.2" PrivateAssets="all" />
```

Comment:
Why no more PrivateAssets? Is this one actually needed somewhere?

Reply:
I added private assets because that's present in new projects, assume it was missed/ missing previously.

Reply:
Ah I see, I had it the wrong way around, good job 👍